src/components/issues/Issue.vue

```diff
@@ -21,11 +21,30 @@
         <span class="issue">#{{ issue.id }} {{ issue.subject }}</span>
         <span class="project">{{ issue.project.name }}</span>
       </v-flex>
-      <v-flex xs1>
+      <v-flex xs2>
         <v-list-tile-action>
-          <v-icon @click="startNewTimer({ issueId: issue.id, comments: '' })">
-            play_circle_outline
-          </v-icon>
+          <v-layout row align-center justify-end>
+            <v-btn
+              class="mr-2"
+              text
+              icon
+              target="_blank"
+              @click="
+                openExternalLink(
+                  $store.state.profile.current.url + 'issues/' + issue.id
```

Comment:
needs to be fixed for tracker URL saved without the slash otherwise, this is the URL generated:

http://domain.itissues/1